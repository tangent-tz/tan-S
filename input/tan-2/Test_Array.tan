main {
    var y := 2; 
    print y \n;
    
    var int_array := [101, -2, 97*10-20, 1+2+2];
    print int_array \n int_array \n "end." \n;
    
    var float_array := [1.1, -2.2222, +3.45, 0.223, -23.3, 1.0+2.2+3.3*4.4];
    print float_array \n; 
    
    var char_array := ['a', 'g', 'h', 't']; 
    print char_array \n; 
    
    const string_array := ["hello", "world", "it's", "me", "!", "helluuuu"]; 
    print string_array \n; 
    
    const bool_array := [true, false, false, true, true]; 
    print bool_array \n; 
    
    print [1, 2, 3] \n; 
    
    print true \n; 
    
    print "hello" \n;
    
}