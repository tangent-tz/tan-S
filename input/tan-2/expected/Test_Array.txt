2
[ 101, -2, 950, 5 ]
[ 101, -2, 950, 5 ]
end.
[ 1.100000, -2.222200, 3.450000, 0.223000, -23.300000, 17.720000 ]
[ a, g, h, t ]
[ hello, world, it's, me, !, helluuuu ]
[ true, false, false, true, true ]
[ 1, 2, 3 ]
true
hello
[ 1, 22, 333 ]
[ 4, 5, 6, 7 ]
[ 4, 5, 6, 7 ]
[ 4, 5, 6, 7 ]
